app/src/main/java/com/nowak/wjw/simplecompass/di/AppContainer.java

```diff
@@ -1,27 +1,57 @@
 package com.nowak.wjw.simplecompass.di;
 
 import android.content.Context;
-import android.hardware.Sensor;
 import android.hardware.SensorManager;
 
-import timber.log.Timber;
+import com.google.android.gms.location.LocationServices;
+import com.nowak.wjw.simplecompass.data.Compass;
+import com.nowak.wjw.simplecompass.domain.GetAzimuthUseCase;
+import com.nowak.wjw.simplecompass.domain.GetDestinationBearingUseCase;
+import com.nowak.wjw.simplecompass.domain.InitiateLastLocationUseCase;
+import com.nowak.wjw.simplecompass.domain.RequestAndStopLocationUpdatesUseCase;
+import com.nowak.wjw.simplecompass.domain.StartStopSensorListenerUseCase;
+import com.nowak.wjw.simplecompass.data.location.LocationApiHandler;
+import com.nowak.wjw.simplecompass.data.sensors.SensorHandler;
+import com.nowak.wjw.simplecompass.ui.main.MainViewModelFactory;
 
 public class AppContainer {
```

Comment:
jakaś forma DI to w sumie jest 
https://developer.android.com/reference/android/app/Application.html